specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -659,6 +698,49 @@
         "queryTimeout": {
           "type": "object",
           "description": "Query timeout. Type: string (or Expression with resultType string), pattern: ((\\d+)\\.)?(\\d\\d):(60|([0-5][0-9])):(60|([0-5][0-9]))."
+        },
+        "partitionOption": {
+          "description": "The partition mechanism that will be used for oracle read in parallel.",
+          "type": "object",
+          "enum": [
+            "None",
+            "PhysicalPartitionsOfTable",
+            "DynamicRange"
+          ],
+          "x-ms-enum": {
+            "name": "TabularPartitionOption",
+            "modelAsString": true
+          }
+        },
+        "partitionSettings": {
+          "description": "The settings that will be leveraged for oracle source partitioning.",
+          "$ref": "#/definitions/OraclePartitionSettings"
+        }
+      }
+    },
+    "OraclePartitionSettings": {
+      "description": "The settings that will be leveraged for oracle source partitioning.",
+      "type": "object",
+      "properties": {
+        "partitionNames": {
+          "type": "array",
```

Comment:
you might want to make the type of this object, so the whole property can be used as an expression instead of array of expressions